check_signed_off.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "Checking range: ${TRAVIS_COMMIT_RANGE}"
+
+NEW_COMMITS=( $(git log  --format=format:%H "$TRAVIS_COMMIT_RANGE") )
+
+for i in "${NEW_COMMITS[@]}"
+do
+    PARENTS=$(git cat-file -p "$i" | grep -c "parent")
+    if [ "$PARENTS" -gt 1 ];
+     then
+        #if the commit is a merge, dont check the commit msg.
+        continue
+     else
```

Comment:
This can also be simplified, just call `git rev-list --no-merges` above, and you will not get any merge commits in the output.